packages/material-ui/src/utils/debounce.js

```diff
@@ -3,10 +3,8 @@
 export default function debounce(func, wait = 166) {
   let timeout;
   function debounced(...args) {
-    // eslint-disable-next-line consistent-this
-    const that = this;
```

Comment:
`babel` adds a reference when transpiling arrow functions. Maybe it didn't used to.

Reply:
I've used Babel since Babel 6 (I think I started in 2016) and it transpiled arrow functions via `var that = this` back then as well. I think it's the only way that it could output ES2015 for older browsers so pretty sure it's not that new.